src/event.js

```diff
@@ -428,4 +428,57 @@ geo_event.camera.projection = 'geo_camera_projection';
 //////////////////////////////////////////////////////////////////////////////
 geo_event.camera.viewport = 'geo_camera_viewport';
 
+////////////////////////////////////////////////////////////////////////////
+/**
+ * These events are triggered by the annotation layer.
+ * @namespace geo.event.annotation
+ */
+////////////////////////////////////////////////////////////////////////////
+geo_event.annotation = {};
```

Comment:
same here verb vs noun? I do not have strong preferences and my comments are more as second thoughts.